src/types/docs-schemas/resources.schema.json

```diff
@@ -58,6 +58,22 @@
               "$ref": "definitions.schema.json/#/definitions/sentinel",
               "format": "uri-reference"
             }
+          },
+          "deployment-configs": {
+            "title": "Deployment Configs",
+            "type": "object",
+            "additionalProperties": {
+              "$ref": "definitions.schema.json/#/definitions/deployment-config",
+              "format": "uri-reference"
+            }
+          },
+          "block-explorer-api-keys": {
+            "title": "Block Explorer Api Keys",
+            "type": "object",
+            "additionalProperties": {
+              "$ref": "definitions.schema.json/#/definitions/block-explorer-api-key",
```

Comment:
Same here, double check if the reference to the definitions schema is correct. I think this might need to be `blockExplorerApiKey`.